docs/DeveloperGuide.adoc

```diff
@@ -1729,7 +1762,7 @@ These instructions only provide a starting point for testers to work on; testers
 
 . Adding an Item into the current Receipt.
 
-.. Add item command format: `additem -i ITEMNAME -p ITEMPRICE -n PERSON [-n PERSON]`
+.. Add Item Command format: `additem -i ITEMNAME -p ITEMPRICE -n PERSON [-n PERSON]`
```

Comment:
I think the last [-n PERSON] should have a spread operator right? like [-n PERSON]...